src/js/modules/infragistics.ui.scroll.js

```diff
@@ -1197,7 +1197,7 @@
 		_linkElementsH: function (inElements) {
 			this._linkedHElems = [];
 			if (inElements) {
-				for (var index in inElements) {
+				for (var index = 0; index < inElements.length; index++) {
```

Comment:
Use the `hasOwnProperty()` check instead. Leave the for-in loop, but wrap everything inside in a hasOwnProperty check:

```
for (var index in inElements) {
    if (inElements.hasOwnProperty(index)) {
        ...
    }
}
```

That should fix your issue.